modules/service/keeper/invocation.go

```diff
@@ -457,16 +456,11 @@ func (k Keeper) buildRequest(
 	batchCounter uint64,
 	serviceName string,
 	provider sdk.AccAddress,
-	superMode bool,
 	consumer sdk.AccAddress,
 	timeout int64,
 ) types.CompactRequest {
-	var serviceFee sdk.Coins
-
-	if !superMode {
-		binding, _ := k.GetServiceBinding(ctx, serviceName, provider)
-		serviceFee = k.GetPrice(ctx, consumer, binding)
-	}
+	binding, _ := k.GetServiceBinding(ctx, serviceName, provider)
```

Comment:
why ignore err ？

Reply:
This is an internal method as described in its comment.